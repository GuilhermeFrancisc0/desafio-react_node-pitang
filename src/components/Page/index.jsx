import React from 'react';
import { Container } from 'react-bootstrap';
import Card from '../Card';

export default function Page({ children, title }) {
  return (
    <Container className="mt-4">
      <Card title={title}>{children}</Card>
    </Container>

  );
}
