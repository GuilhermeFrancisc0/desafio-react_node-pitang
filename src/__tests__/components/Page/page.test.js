/* eslint-disable no-undef */
import React from 'react';

import { render } from '@testing-library/react';
import Page from '../../../components/Page';

test('Page renders', () => {
  const { asFragment } = render(<Page />);

  expect(asFragment()).toMatchSnapshot();
});
