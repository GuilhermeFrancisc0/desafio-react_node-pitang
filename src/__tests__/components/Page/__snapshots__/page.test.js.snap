// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page renders 1`] = `
<DocumentFragment>
  <div
    class="mt-4 container"
  >
    <div
      class="card"
    >
      <div
        class="card-header"
      >
        <div
          class="card-title h5"
        />
      </div>
      <div
        class="card-body"
      />
    </div>
  </div>
</DocumentFragment>
`;

exports[`renders correctly 1`] = `
<DocumentFragment>
  <div
    class="mt-4 container"
  >
    <div
      class="card"
    >
      <div
        class="card-header"
      >
        <div
          class="card-title h5"
        />
      </div>
      <div
        class="card-body"
      />
    </div>
  </div>
</DocumentFragment>
`;
